content/en/containers/guide/cluster_agent_autoscaling_metrics.md

```diff
@@ -425,6 +442,37 @@ The four conditions give you insights on the current state of your `DatadogMetri
 
 The `currentValue` is the value gathered from Datadog and returned to the HPAs.
 
+### Value vs AverageValue for the target metric
+The HPAs in these example use the target type of `Value` instead of `AverageValue`. Either option is supported, however, you adjust your Datadog metric queries accordingly.
```

Comment:
```suggestion
The HPAs in these example use the target type of `Value` instead of `AverageValue`. Both options are supported. Adjust your Datadog metric queries accordingly.
```